src/r_draw.c

```diff
@@ -72,6 +74,7 @@ static pixel_t *background_buffer = NULL;
 //
 
 lighttable_t *dc_colormap[2]; // [crispy] brightmaps
+lighttable_t *dc_tint; // thanks to Doom Retro
```

Comment:
Just like with the `// [EA]` comments previously, we generally don't add comments like this unless retained for historical purposes.